src/legacy/js/functions/_t4BulletinEditor.js

```diff
@@ -180,7 +180,16 @@ function bulletinEditor(collectionId, data) {
             newFiles[indexF] = {title: title, file: file};
         });
         data.pdfTable = newFiles;
-
+        
+        // tags
+        if ($("#selectTopic").val() && $("#selectSubTopic").val()) {
+            data.description.primaryTopic = $("#selectTopic").val()[0]
```

Comment:
Just a question: As long as you're sure `$("#selectTopic").val()` is always going to be an array LGTM